jimbopulos/ecothot
# Setup Local Supabase Instance and Configuration

## Addresses

[Setup Supabase local PSQL db mgmt](https://app.asana.com/0/1202639946826739/1207048805583402/f)

## Summary
- Initializes local Supabase instance setup
- Moves `/supbase` dir to root
- Updates `README.md`

## Testing
- Run `npm install` from root
- Make sure you have Docker and [Docker Desktop](https://www.docker.com/products/docker-desktop/) installed
- Then, run `supabase start` (if you don't have `supabase` installed globally, you'll probably have to prefix the command): `npx supabase start`
- You should see the Docker image begin to build
- When complete, the output like:
  ```bash
  Started supabase local development setup.

  API URL: http://127.0.0.1:...
  GraphQL URL: http://127.0.0.1:...
  S3 Storage URL: http://127.0.0.1:54321/storage/v1/s3
  DB URL: postgresql://postgres:postgres@127.0.0.1:54322/...
  ...
  ```
- You can confirm in Docker Desktop that there is an `ecothot` image with status Running
